utils/githooks/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# A git hook to validate  and correct the copyright date in source files.
+
+regex='(^\s*\*.*)((Copyright\s*)([0-9]{4})(-([0-9]{4}))?)(\s*(Intel.*$))'
+year=$(date +%Y)
+errors=0
+
+files=$(git diff --diff-filter=AM --name-only --cached -- '*.c' '*.h' 'Makefile')
```

Comment:
This means copyright fixing is going to be limited to `*.c`,`*.h`, and `Makefile`s doesn't it?

Why such a limit?